### PR TITLE
[make:security:form-login] no interaction for make:security:form-login

### PR DESCRIPTION
**Pull request by eltharin**
allow command make:security:form-login with name in argument to no interaction

as help says : make:security:form-login \[\<controllerName\>\]

### CODE REVIEW

File: src/Maker/Security/MakeFormLogin.php [hunk 1]
```diff
@@ -57,11 +59,11 @@ final class MakeFormLogin extends AbstractMaker
 
     private const SECURITY_CONFIG_PATH = 'config/packages/security.yaml';
     private YamlSourceManipulator $ysm;
-    private string $controllerName;
     private string $firewallToUpdate;
     private string $userClass;
     private string $userNameField;
-    private bool $willLogout;
```
Comment (jrushlow) [inline src/Maker/Security/MakeFormLogin.php:64]: We need to keep both of these properties. MakerBundle is getting away from "abusing" `setArgument` and `getArgument` for the purposes of passing user input from one method to the next within a maker command. e.g. separation of concerns. 

tl;dr `interact()` configures and sets any values onto their respective properties that will later be needed in `generate()`

`interact()` should get, validate, and set input.
`generate()` should only use values that are set via class properties when possible.

File: src/Maker/Security/MakeFormLogin.php [hunk 2]
```diff
@@ -111,38 +116,44 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
...
+                    'SecurityController',
+                    Validator::validateClassName(...)
+                ));
+        }
 
-        $securityHelper = new InteractiveSecurityHelper();
-        $this->firewallToUpdate = $securityHelper->guessFirewallName($io, $securityData);
-        $this->userClass = $securityHelper->guessUserClass($io, $securityData['security']['providers']);
-        $this->userNameField = $securityHelper->guessUserNameField($io, $this->userClass, $securityData['security']['providers']);
-        $this->willLogout = $io->confirm('Do you want to generate a \'/logout\' URL?');
```
Comment (jrushlow) [inline src/Maker/Security/MakeFormLogin.php:131]: Following on from the above, these all need to stay in interact. They should not be set in the `generate` method.

**Review by jrushlow — CHANGES_REQUESTED** [review]
A couple of comments and we can get this moving...

File: src/Maker/Security/MakeFormLogin.php (hunk 1 above)
Comment (eltharin) [reply src/Maker/Security/MakeFormLogin.php:64]: ok so we don't have to use --no-interaction option for all makers ?

**Comment by eltharin** [comment]
@jrushlow Is this as you want now ?
